src/pybind/mgr/dashboard/services/ganesha.py

```diff
@@ -118,7 +124,7 @@ def make_rados_url(cls, pool, namespace, obj):
 
     @classmethod
     def get_pool_and_namespace(cls, cluster_id):
-        instances = cls._get_orch_nfs_instances()
+        instances = cls._get_orch_nfs_services()
         # we assume that every instance stores there configuration in the
         # same RADOS pool/namespace
         if instances:
```

Comment:
Is the assumption still valid? We now can provision multiple NFS services, using the first pool/namespace pair will make others unavailable.

BTW, can we remove orchestrator calls from the Ganesha controller since Ganesha daemons now watch rados objects? (The reload function doesn't work at all and no one seems complain so far...)

Reply:
@jtlayton @varshar16 : Do you know if reloading the ganesha daemon is still required? If yes, when is it required?

Reply:
Even though Ganesha daemon watch rados objects, they need to be notified about any changes in configuration. So don't remove this reload method from `GaneshaConf` class and it did not work prior to NFS-Ganesha v3.3.
https://github.com/ceph/ceph/blob/da9f387e6a9888ab58416fc734715a5672e1160a/src/pybind/mgr/dashboard/services/ganesha.py#L997-L1002



But we can remove this reload method from `Ganesha` class which reloads the service and I don't see it being used anywhere.
https://github.com/ceph/ceph/blob/da9f387e6a9888ab58416fc734715a5672e1160a/src/pybind/mgr/dashboard/services/ganesha.py#L135-L150